interfaces/Portal/src/app/auth/auth.service.ts

```diff
@@ -12,7 +12,7 @@ import Permission from './permission.enum';
 export class AuthService {
   public redirectUrl: string;
   public isIframe: boolean;
-  private userKey = 'logged-in-user-HO';
+  private userKey = 'logged-in-user-portal';
```

Comment:
@jannisvisser This will log-out all currently logged-in users. 

Not the end of the world, but still might be good to warn/notify users.

Reply:
@elwinschmitz made a note in release item